.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
```

Comment:
Report section

Reply:
If you can add a Report section then if users are unable to use any feature then can report to the developer team